infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -2,24 +2,19 @@
 x-defaults: &default-release-image-source
   releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
   releaseImageName: dpl-cms-source
-  dpl-cms-release: "2024.50.0"
+  dpl-cms-release: "2025.3.2"
 x-webmaster: &webmaster-release-image-source
+  # This is the default release plan for webmasters. There's currently no webmasters on it.
   releaseImageRepository: ghcr.io/danskernesdigitalebibliotek
   releaseImageName: dpl-cms-source
-  dpl-cms-release: "2024.49.0"
-  moduletest-dpl-cms-release: "2024.50.0"
-# Some sites on the webmaster plan wish to track the same release as
-# sites do per default in dpl-cms-release.
-x-webmaster-with-defaults: &webmaster-with-defaults-release-image-source
-  <<: *default-release-image-source
-  moduletest-dpl-cms-release: "2024.50.0"
-# Postponed 2024.12.12, they don't want 2024.49.0 on production,
-# postponing it to 2024.50.0.
-x-webmaster-postponed-until-2025-01-06: &webmaster-postponed-until-2025-01-06
+  dpl-cms-release: "2024.50.0"
+  moduletest-dpl-cms-release: "2025.3.2"
```

Comment:
Consider adding more details on this release plan (last week of every month?) people know when to update this.